examples/misc/raptor.py

```diff
@@ -47,52 +77,58 @@ def task_state_cb(task, state):
 if __name__ == '__main__':
 
     if len(sys.argv) < 2:
-        cfg_file = './raptor.cfg'
+        cfg_file = os.path.join(os.path.dirname(__file__), 'raptor.cfg')
     else:
         cfg_file = sys.argv[1]
 
     cfg         = ru.Config(cfg=ru.read_json(cfg_file))
     sleep       = int(cfg.sleep)
-    cpn         = cfg.cpn
-    gpn         = cfg.gpn
+    cores_per_node         = cfg.cores_per_node
+    gpus_per_node         = cfg.gpus_per_node
     n_masters   = cfg.n_masters
     n_workers   = cfg.n_workers
-    masters_pn  = cfg.masters_pn
-    nodes_pw    = cfg.nodes_pw
+    masters_per_node  = cfg.masters_per_node
+    nodes_per_worker  = cfg.nodes_per_worker
     nodes_rp    = cfg.nodes_rp
     workload    = cfg.workload
     tasks_rp    = cfg.tasks_rp
     nodes_agent = cfg.nodes_agent
 
+    cores_per_task = cfg.cores_per_task
+
     # each master uses a node, and each worker on each master uses a node
     # use 8 additional cores for non-raptor tasks
```

Comment:
Is this two sentences or one sentence with extra words?
I'm not sure what is meant, what determines these values, and to what degree they are configurable.

Reply:
Thanks - I clarified the comment.